tests/e2e/env/setup.sh

```diff
@@ -161,6 +161,12 @@ cli wp wc --user=admin tool run install_pages
 echo "Importing some sample data..."
 cli wp import wp-content/plugins/woocommerce/sample-data/sample_products.xml --authors=skip
 
+echo "Adding customer account ..."
```

Comment:
In the `createCustomerUser` function, we were always deleting first and then creating, for cleanups from previous tests I presume. Do we need to do that here across runs since this will be at the setup level?

Reply:
@deepakpathania Sure we can change the order. That would solve the error you got if we re-run the setup script on top of the existing env.

Reply:
Done